src/bundle_remove.c

```diff
@@ -89,11 +93,389 @@ static bool parse_options(int argc, char **argv)
 		return false;
 	}
 
+	/* flag restrictions */
+	if (cmdline_option_force && cmdline_option_recursive) {
+		error("--force and --recursive options are mutually exclusive\n\n");
```

Comment:
With this new implementation of recursive, why they need to be mutually exclusive?
Force will create a list of all bundles including dependencies. And then --recursive will select all dependencies in the system that are leftovers that can be removed.
And your code is already supporting that.

Reply:
well it may be supported but using them together makes the command even more dangerous to be used, and more probable something does't work as expected in a corner case since we add more variables to the equation. I don't know if there is a use case for wanting to run both together. Having said that I'm ok to remove the restriction if you think we should support that. Do you think we should support that?

Reply:
Both --force and --recursive are dangerous. Even isolated. So I'm ok on adding that support.
I wonder if we should add a warning on both --force and --recursive that this should be run with caution.

Reply:
I will remove the restriction then.
We already print these messages when using those flags:
```
	/* inform user about the effects of using the options */
	if (cmdline_option_force) {
		info("\nThe --force option was used, the specified bundle%s and all bundles "
		     "that require %s will be removed from the system\n",
		     total > 1 ? "s" : "",
		     total > 1 ? "them" : "it");
	} else if (cmdline_option_recursive) {
		info("\nThe --recursive option was used, the specified bundle%s and %s "
		     "dependencies will be removed from the system\n",
		     total > 1 ? "s" : "",
		     total > 1 ? "their" : "its");
	}
```
I can use `warn` instead of `info`

Reply:
No info is fine for that. But when the user sees it it's a bit too late. haha
Let's add a warning in the manual and help messages.

Reply:
done, it will be in the next update